client/components/Muncher/Muncher.js

```diff
@@ -1,15 +1,65 @@
 import { useState } from 'react';
 import { useUser } from '../../context/UserContext.js';
+import { useMuncher } from '../../hooks/useMuncher.js';
 import { enforceUser } from '../../services/UserService.js';
+import SpeechToText from '../SpeechToText/SpeechToText.js';
 import styles from './Muncher.css';
 
 export default function Muncher() {
   const { user, loading } = useUser();
   enforceUser(user, loading);
 
-  const [game, setGame] = useState();
+  const { 
+    game,  
+    targetSound, setTargetSound, 
+    points, 
+    lives, 
+    gameover,
+    resetGame,
+    currentPosition,
+    handleMove,
+    handleEat,
+  } = useMuncher();
+
+  if (gameover) {
+    return (
+      <div>
+        <h2 className={'title'} >
+          Game Over!
+        </h2>
+        <h2>total points: {points}</h2>
+        <div>
+          <button onClick={resetGame}>
+            Play again?
+          </button>
+        </div>
+      </div>
+    );
+  }
```

Comment:
I might recommend moving this gameover return into its own component, just for readability. This is personal preference though, so it's really up to you!